scripts/flattener.py

```diff
@@ -382,15 +384,15 @@ def report_dataset(donor_objs, matrix, dataset):
 	derived_by = matrix.get('derivation_process')
 
 	# NEED TO WORK OUT LOGIC FOR MULTIPLE LAYERS, RECONCILING WHEN FINAL MATRIX IS SEURAT VS H5AD
-	x_norm = ""
+	#x_norm = ""
```

Comment:
A lot of this function can removed. I believe lines 381 (`layer_descs = {`) to line 413 (`ds_results['X_normalization'] = x_norm`) can all be removed. We don't need anything regarding "layers_desc".  This will need to be tested.